README.md

```diff
@@ -19,8 +19,9 @@ successfully tested on the following boards:
 
```

Comment:
Do you want to mention the DFU-based boards in this README? And the DFU limitations regarding re-flashing Tock?